run_container.sh

```diff
@@ -51,10 +44,10 @@ while [[ $# -gt 0 ]]; do
 	shift
 done
 
-# if both -d and -i args are passed, run the flasher script in container directly
-if [ ! -z ${dram_conf+x} ] && [ ! -z ${balena_image+x} ]; then
+# if -i arg is passed, run the flasher script in container directly
+if [ [ ! -z ${balena_image+x} ]; then
```

Comment:
@vraevsky apart from this small typo, this PR looks good to me

Reply:
and the typo is now fixed in https://github.com/balena-os/iot-gate-imx8plus-flashtools/pull/9

Reply:
@acostach I'd appreciate it if you could point me to the typo.

Reply:
Extra bracket